com.unity.multiplayer.mlapi/Tests/Editor/NetworkObjectTests.cs

```diff
@@ -0,0 +1,46 @@
+using NUnit.Framework;
+using UnityEngine;
+using UnityEngine.TestTools;
+
+namespace MLAPI.EditorTests
+{
+    public class NetworkObjectTests
+    {
+        [Test]
+        public void GetBehaviourIndexNone()
+        {
+            var gameObject = new GameObject(nameof(GetBehaviourIndexNone));
```

Comment:
I believe we should destroy this gameObject since its scope looks super local and specific to this test.